utils/validate_release_links.py

```diff
@@ -85,7 +85,7 @@ def update_pip_install_line(line, package_verion):
     match = PIP_INSTALL_PATTERN.search(line)
     if match is not None:  # if there is a pip install line
         package_name = match.group("package")
-        quiet_option = match.group("quiet")
+        quiet_option = match.group("quiet") or ""
```

Comment:
Maybe there's a better way to do this with the capture in the regex, but I think this works too.